src/IssueViz/IssueVisualizationControl/IssueVisualizationControl.xaml

```diff
@@ -242,159 +242,13 @@
-                            <ContentPresenter Name="ContentSite" 
-                                              IsHitTestVisible="False"
-                                              HorizontalAlignment="Left" 
-                                              VerticalAlignment="Center"          
-                                              Margin="{TemplateBinding Margin}" 
-                                              Content="{TemplateBinding ComboBox.SelectionBoxItem}" 
-                                              ContentTemplate="{TemplateBinding ComboBox.SelectionBoxItemTemplate}"
-                                              ContentTemplateSelector="{TemplateBinding ItemTemplateSelector}"/>
-                            <Popup Name="Popup" 
-                                   Placement="Bottom"
-                                   Focusable="False" 
-                                   AllowsTransparency="True"
-                                   IsOpen="{TemplateBinding ComboBox.IsDropDownOpen}"
-                                   PopupAnimation="Slide">
-                                <Grid Name="DropDown" 
-                                      SnapsToDevicePixels="True"
-                                      MinWidth="{TemplateBinding FrameworkElement.ActualWidth}"
-                                      MaxHeight="{TemplateBinding ComboBox.MaxDropDownHeight}">
-                                    <Border Name="DropDownBorder" 
-                                            Background="{DynamicResource {x:Static vsShell:VsBrushes.ComboBoxBackgroundKey}}" 
-                                            Margin="0,1,0,0"
-                                            CornerRadius="0" 
-                                            BorderThickness="1" 
-                                            BorderBrush="{DynamicResource {x:Static vsShell:VsBrushes.ComboBoxBorderKey}}"/>
-                                    <ScrollViewer Margin="4" SnapsToDevicePixels="True">
-                                        <StackPanel IsItemsHost="True" KeyboardNavigation.DirectionalNavigation="Contained" />
-                                    </ScrollViewer>
-                                </Grid>
-                            </Popup>
-                        </Grid>
-                        <ControlTemplate.Triggers>
-                            <Trigger SourceName="Popup" Property="Popup.AllowsTransparency" Value="true">
-                                <Setter TargetName="DropDownBorder" Property="CornerRadius" Value="0"/>
-                                <Setter TargetName="DropDownBorder" Property="Margin" Value="0,2,0,0"/>
-                            </Trigger>
-                        </ControlTemplate.Triggers>
-                    </ControlTemplate>
-                </Setter.Value>
-            </Setter>
+        <Style x:Key="FlowsComboBox" TargetType="{x:Type ComboBox}" BasedOn="{StaticResource {x:Static vsShell:VsResourceKeys.ComboBoxStyleKey}}">
```

Comment:
I think this is the missing piece we were looking for: `vsShell:VsResourceKeys` has a number of styles defined.

Reply:
Nicely done; I hope we can apply it to the datagrid and context menu controls as well (that we use in hotspots and taint).